covid19-detection/build.gradle

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id "java"
+    id "application"
+}
+
+group "com.example"
+version "1.0-SNAPSHOT"
+
+sourceCompatibility = 1.8
+
+repositories {
+    mavenCentral()
+    maven {
+        url "https://oss.sonatype.org/content/repositories/snapshots/"
+    }
+}
+
+dependencies {
+    implementation "ai.djl:api:0.5.0-SNAPSHOT"
```

Comment:
Can this target 0.4.x so it doesn't break with a later version release?

Reply:
We don't have 0.4.x release for tensorflow api and native jar